packages/admin/Dockerfile

```diff
@@ -45,6 +45,9 @@ COPY --chown=node:node . .
 #
 FROM base as develop
 
+# Check every 60s to ensure this service returns HTTP 200
+HEALTHCHECK --interval=60s CMD node healthcheck.js
```

Comment:
I have refactored for a single call like you suggested with 
docker compose ps -q | xargs docker inspect --format='{{.State.Health.Status}}' | grep -q 'unhealthy';

I have also added a start-period=10 seconds to the dockerfile health checks, so that it will wait ten seconds before the initial check instead of 60 as it was doing. Now in the action it will wait 15s, check for health, then wait once for 60 seconds before finishing. So far the services are up and healthy for the first check